packages/user/src/User/User.purs

```diff
@@ -167,6 +171,38 @@ createUserWithEmail newTemporaryUser = do
           pure $ Left $ UnexpectedError err
     Right user -> finalizeLogin Nothing =<< saveToken user
 
+createCusnoUser :: Persona.NewCusnoUser -> Aff (Either UserError SearchResult)
+createCusnoUser newCusnoUser = do
+  newUser <- try $ Persona.registerCusno newCusnoUser =<< requireToken
+  case newUser of
+    Left err
+      | Just (errData :: Persona.EmailAddressInUseRegistration) <- Api.Error.errorData err -> do
+          Console.error errData.email_address_in_use_registration.description
```

Comment:
Do we want to send any of these errors to Sentry?

Reply:
Maybe? I don't know, any users of this function would be internal users and if they see something wrong they'd know to contact us.

Reply:
Fair enough. Don't know if logging to the console is of any use though 🤔

Reply:
It's to us if something goes wrong and we try to debug it.

Reply:
Well, they are useful only for developers and only while developing the app. If they were log messages that actually needed our attention, the correct place to send them would be Sentry.

Reply:
And I'm not denying their usefulness for developers. Just that in production they're basically just noise.